### ISSUE TITLE
[PLUGIN] Is there a way to attach a new external css to my custom plugin ?

### ISSUE THREAD

**Issue by RaspFR** [Xzandro/sw-exporter#279]
Hi Xzandro, 
I've developped a swex plugin about guild ranking that acutally print in the log a table of all my guildies sorted by overall contribution. (see screenshot attached). 

I can make it as beautiful as I want because I was unable to attached to my plugin js file an external css that I've created in the same folder. And I don't want to inject complex css in every html style attribute...

So my question is : Is there a way to do it or do you have to improve the swex plugins system to make it possible ?

![contrib](https://user-images.githubusercontent.com/10197265/64569043-9d3d4100-d35c-11e9-8de9-6e558a6a763a.png)

Thanks a lot for your great work ;)

**Comment by Xzandro** [comment]
I don't think you can utilize additional CSS other than what's provided by semantic-ui. Your best bet is probably dump a custom HTML like file and do all styling there, or create an external website and post all your data to this service. The Log section isnt really made to post in highly complex and styled data.

**Comment by Artenuvielle** [comment]
I'd suggest exposing the BrowserWindow class from electron to plugins as third parameter for init as done in #305.
This way plugins can handle their own windows for display and have links to css files in there.

**Comment by Artenuvielle** [comment]
While providing the BrowserWindow class to the plugins from SWEX would work you can also get the constructor for BrowserWindow from the globally registered 'win' variable which holds the electron main window of SWEX. Since its constructor is not accessible by the instance itself you have to access __proto__ for that. Example:
```
    new global.win.__proto__.constructor({  // BrowserWindow constructor options here
        x: 100,
        y: 100,
        minWidth: 1800,
        minHeight: 906
    });
```
